pages/chefs.tsx

```diff
@@ -2,15 +2,54 @@ import Sidebar from "@/shared/components/Sidebar/Sidebar";
 import React from "react";
 import GeneralTable from "../shared/components/GeneralTable/GeneralTable";
 import { chefColumns } from "../data/tableColumns.data";
-import { getChefs } from "@/services/chef.service";
 import { IChef } from "@/models/chef.model";
 import { HttpClientService } from "@/services/HttpClient.service";
+import ActionButton from "@/shared/components/ActionButton/ActionButton";
+import AddIcon from "@mui/icons-material/Add";
 
 const Chefs = ({ chefsData }: { chefsData: IChef[] }) => {
+  function handleCreateNew(): void {
+    throw new Error("Function not implemented.");
+  }
+
+  const handleEdit = (rowData: IChef) => {
+    console.log("Editing chef:", rowData);
```

Comment:
remove log. if not use the func comment it and add comment as. 
`// TODO :  create a edit func`